### ISSUE TITLE
EXR export (Blender) … material greyscale slots export to RGB4 instead of monochrome image. 

### ISSUE THREAD

**Issue by leonard7e** [RodZill4/material-maker#338]
**Material Maker version:**

OS: Ubuntu 20.04
GPU: RTX 2060 (proprietary NVidia drivers)

**Issue description:**
MM could export EXR files (through Blender export).

Monochrome material slots are not exported to monochrome EXR, but to RGBA. Makes usage of monochrome EXR output rather counter intuitive and clunky. 
It also might be that R slot is not used. It also inflates EXR file size.

Reproducing is simple: just plug a pattern node to a monochrome Material slot.

**Comment by johannes-wilde** [comment]
This is what I realized as well yesterday.

PopOS 21.04
Material Maker 0.97

**Comment by RodZill4** [comment]
I just tried to fixed this. Please don't hesitate to grab the latest dev build and tell me if it solves the problem for you.

**Comment by leonard7e** [comment]
I used 'Blender' export to generate EXR files.

EXR files are much smaller! And it looks like greyscale images are single-slot now. In Blender a MM generated displace exr image is shown as greyscale now.

Testing a displace file with exrheader …
```
leonard@LuckDragon ~/D/mm_exr> exrheader test_displace.exr 
```
reveals … only one 16 bit channel used, and the image data is compressed (PIZ)

It looks like 2 issues (large file size and monochrome image) has been fixed.

I am glad that you helped with this issue. Thank you.
